ddtrace/llmobs/_llmobs.py

```diff
@@ -1532,7 +1557,20 @@ def _activate_llmobs_distributed_context(cls, request_headers: Dict[str, str], c
         except ValueError:
             log.warning("Failed to parse LLMObs parent ID from request headers.")
             return "invalid_parent_id"
+        _llmobs_trace_id = context._meta.get(PROPAGATED_LLMOBS_TRACE_ID_KEY)
+        if _llmobs_trace_id is None:
+            log.warning("Failed to extract LLMObs trace ID from request headers.")
```

Comment:
can we explicitly say that the trace id is None here? Also this might be pretty noisy since it's expected if there is no distributed tracing enabled.  

```suggestion
            log.debug("Failed to extract LLMObs trace ID from request headers.")
```

Reply:
(the log above this one about the parent id is also noisy lol)